scripts/test_trading_config_migration.py

```
@@ -0,0 +1,316 @@
+    print("=" * 80)
+    print()
+    
+    user_id = "lian"  # 기본 테스트 사용자
+    
+    try:
+        current_user_id = get_current_user_id()
+        if current_user_id:
+            user_id = current_user_id
+            print(f"📌 현재 사용자 ID: {user_id}")
+            print()
+    except:
```

Ruff (0.14.10)

Do not use bare `except`

(E722)